package.json

```diff
@@ -34,8 +34,8 @@
     "tether": "^1.3.7"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0"
+    "react": "^16.0.0",
```

Comment:
Support for 0.14 and 15.x probably shouldn't be removed

Reply:
@rafeememon ok will fix this ASAP.